src/reuse/_util.py

```diff
@@ -222,8 +225,8 @@ def make_copyright_line(
     copyright_prefix = _COPYRIGHT_STYLES.get(copyright_style)
     if copyright_prefix is None:
         raise RuntimeError(
-            "Unexpected copyright style: Need 'spdx', 'spdx-symbol', 'string', 'string-c',"
-            "'string-symbol' or 'symbol'"
+            "Unexpected copyright style: Need 'spdx', 'spdx-symbol', 'string',"
+            "'string-c', 'string-symbol' or 'symbol'"
```

Comment:
missing space between 'string' and 'string-c': `'string,'string-c', 'string-symbol'`

Reply:
Good point!  I went with the same style that was there before and replicated the missing space.  :facepalm:  Fixed in 425904b08f74b16dc6b4e93c503e0cdda777df35.